libraries/Microsoft.Bot.Builder/Teams/TeamsActivityExtensions.cs

```diff
@@ -6,20 +6,38 @@
 
 namespace Microsoft.Bot.Builder.Teams
 {
+    /// <summary>
+    /// The TeamsActivityExtensions
+    /// provides helpers to make interacting with Microsoft Teams objects easier. 
+    /// </summary>
     public static class TeamsActivityExtensions
     {
+        /// <summary>
+        /// Gets the Team's channel id off the current activity.
+        /// </summary>
+        /// <param name="activity"> this activity. </param>
+        /// <returns>current activity's team's channel, or empty string.</returns>
```

Comment:
"The current activity's teams' channel..."

Reply:
updated